CHANGES.md

```diff
@@ -1,3 +1,8 @@
+# 3.0.0 UNRELEASED
+- [FIXED] Expose `BasePlugin` type in Cloudant client.
+- [REMOVED] Remove nodejs-cloudant TypeScript type definitions for `db.search`.
+- [UPGRADED] Using nano==7.1.0 dependancy.
```

Comment:
depend**a**ncy -> depend**e**ncy

Also is it going to be 7.1.0 or do we need something newer with your PRs in?

Reply:
Whoops, excuse the typo. Yeah, I'll bump to 7.1.1. That's what's now in `package.json`.